azure-vote/azure-vote/main.py

```diff
@@ -26,16 +26,11 @@
     title = app.config['TITLE']
 
 # Redis configurations
-redis_server = os.environ['REDIS']
+redis_server = 'azure-vote-back'
 
 # Redis Connection
 try:
-    if "REDIS_PWD" in os.environ:
-        r = redis.StrictRedis(host=redis_server,
-                        port=6379, 
-                        password=os.environ['REDIS_PWD'])
-    else:
-        r = redis.Redis(redis_server)
+    r = redis.StrictRedis(redis_server)
```

Comment:
Again, in regards to front end portability we had included options for including a password. Any particular reason you see to remove this?

Reply:
My thinking was since it wasn't part of the tutorial, I thought it was unused. 

But if this project has another use case I totally understand the desire to set password.

Reply:
Yeah, I think we should keep it. I'm not sure where else in the Azure Samples / Docs / Training ecosystem the application is being used. I would like to keep it generalized so that it can be used in multiple projects if desired.